lib/service/service_test.go

```diff
@@ -1866,6 +1866,8 @@ func TestAgentRolloutController(t *testing.T) {
 	cfg.DebugService.Enabled = false
 	cfg.Auth.StorageConfig.Params["path"] = dataDir
 	cfg.Auth.ListenAddr = utils.NetAddr{AddrNetwork: "tcp", Addr: "127.0.0.1:0"}
+	// Speed up the reconciliation period from 1 minute to 200ms for testing purposes
+	cfg.Auth.AgentRolloutControllerSyncPeriod = 200 * time.Millisecond
```

Comment:
Could we make it faster or will it not make a difference?

Reply:
I can reduce it to 100ms, I'm afraid it might cause some load if we go lower. I used 200ms because this is the same test period we use for our inventory controller tests.

Reply:
If you think it's good do 100ms, otherwise feel free to leave as-is.